circle.yml

```diff
@@ -19,3 +22,18 @@ test:
       zip -r ${CIRCLE_ARTIFACTS}/framework.zip TangramMap.framework; fi
     - if [ $CIRCLE_BRANCH = 'master' ]; then cd build/ &&
       zip -r ${CIRCLE_ARTIFACTS}/docs.zip ios-docs; fi
+deployment:
+  snapshots:
+    branch: [master, deploy-ios-framework]
+    commands:
+      # Create a zip archive of files that will serve as the Cocoapod source.
+      - zip -r ${CIRCLE_ARTIFACTS}/pod.zip build/ios-framework-universal/Release/TangramMap.framework LICENSE
+      - aws s3 cp ${CIRCLE_ARTIFACTS}/pod.zip s3://ios.mapzen.com/tangram-es-snapshots/${CIRCLE_BUILD_NUM}.zip
```

Comment:
A little bit, but for Android we push snapshots at the same frequency and it hasn't been an issue. We actually used to do this for Android on *every* branch, which created a bonkers number of snapshots and I never heard complaints from our AWS bill.